feature/amazon/build.gradle

```diff
@@ -27,5 +27,5 @@ android {
 dependencies {
     implementation project(":purchases")
 
-    implementation libs.amazon.appstore.sdk
+    // implementation libs.amazon.appstore.sdk
```

Comment:
Version 3.0.5 of the amazon appstore sdk if not yet available on Maven Central, so we need to import the .aar manually

https://central.sonatype.com/artifact/com.amazon.device/amazon-appstore-sdk